databases/week3/databases_week3.sql

```diff
@@ -158,13 +161,12 @@ WHERE price < 90;
 -- Get meals that still has available reservations
 
 SELECT meal.id AS meal_id, meal.title AS meal_title,  
```

Comment:
Yep, the left join is the way to go :)

Now to deal with the null values, check the COALESCE() function, it should simplify your query: https://dev.mysql.com/doc/refman/8.0/en/comparison-operators.html#function_coalesce